### PR TITLE
DOC: Updated release notes for 0.20.1

### PR DESCRIPTION
**Pull request by TomAugspurger**
Also added release note for https://github.com/pandas-dev/pandas/pull/16250

### CODE REVIEW

**Comment by TomAugspurger** [comment]
Here's the real diff:

> This is a combined release for 0.20.0 and and 0.20.1.
 Version 0.20.1 contains one additional change for compatibility with downstream projects using pandas' hashing routines. (:issue:`16250`)

Other than that, just renaming the links from 0200 to 0201.

File: doc/source/whatsnew/v0.20.0.txt [hunk 1]
```diff
@@ -1,45 +1,48 @@
-.. _whatsnew_0200:
+.. _whatsnew_0201:
 
-v0.20.0 (May 4, 2017)
+v0.20.1 (May 5, 2017)
 ---------------------
 
+This is a combined release for 0.20.0 and and 0.20.1.
```
Comment (jreback) [inline doc/source/whatsnew/v0.20.0.txt]: maybe make this a ::note (and move to before the toc)?

File: doc/source/whatsnew/v0.20.0.txt [hunk 2]
```diff
@@ -1,45 +1,48 @@
...
+Version 0.20.1 contains one additional change for backwards-compatibility with downstream projects using pandas' hashing routines. (:issue:`16250`)
```
Comment (jorisvandenbossche) [inline doc/source/whatsnew/v0.20.0.txt]: to be correct it are the utils routines, not hashing (the hashing you asked before to add a deprecation shim, which we did)

Comment (jreback) [inline doc/source/whatsnew/v0.20.0.txt]: agreed its really about the utils things.

**Review by jorisvandenbossche — APPROVED** [review]

File: doc/source/whatsnew/v0.20.0.txt [hunk 3]
```diff
@@ -388,8 +393,7 @@ For example, after running the following, ``styled.xlsx`` renders as below:
    df
    styled = df.style.\
        applymap(lambda val: 'color: %s' % 'red' if val < 0 else 'black').\
```
Comment (jreback) [inline doc/source/whatsnew/v0.20.0.txt]: I *think* you have a cleaner way of doing this?

Comment (jorisvandenbossche) [inline doc/source/whatsnew/v0.20.0.txt]: I don't think we have a built-in fucntion for negative values (only for max, min or null)

Comment (TomAugspurger) [inline doc/source/whatsnew/v0.20.0.txt]: I think this is the best we have right now. A `.where` would be nice... I'll make a new issue :)

**Comment by TomAugspurger** [comment]
Doc build finished, so merging.

**Pull request merged**